translations/nl-be.json

```diff
@@ -1354,6 +1354,17 @@
   "check-documents-to-be-sent-to-vp": "Vink de documenten aan die je naar het Vlaams Parlement wilt doorsturen.",
   "possible-pieces-missing": "Kijk na of de bovenstaande lijst volledig is.",
   "automatic-check-performed-possible-pieces-missing":"Op basis van een automatische controle lijken de volgende documenten te ontbreken. Wil je ze toch indienen? Laad ze dan op en stuur ze vervolgens door.",
+  "case-info": "Dossier info",
+  "design-of": "Ontwerp van",
+  "last-statusses": "Laatste statussen",
+  "to-flemish-parliament": "Naar Vlaams Parlement",
+  "number": "Nummer",
+  "accepted-in-plenary-meeting-on": "Aangenomen in plenaire vergadering op",
+  "ratified-and-announced-on": "Bekrachtigd en afgekondigd op",
+  "published-in-belgian-official-gazette": "Bekendgemaakt in het Belgisch Staatsblad op",
+  "sent-on-date-by-person": "Verzonden op {date} door {person}",
+  "no-statusses-available-yet": "Er zijn nog geen statussen beschikbaar",
+  "no-documents-submitted-to-vp-yet": "Er werden nog geen documenten ingediend bij het Vlaams Parlement",
```

Comment:
"doorgestuurd naar"? Not fully in the loop of naming here, but I know there was a discussion between these 2 terms

Reply:
Not sure, there is no design for this, feel free to change.